### PR TITLE
Release the display text texture

### PR DESCRIPTION
**Pull request by RodrigoHolztrattner**
Release it on the display destructor

### CODE REVIEW

File: obs-studio-server/source/nodeobs_display.cpp [hunk 1]
```diff
@@ -361,6 +361,11 @@ OBS::Display::~Display()
 	obs_enter_graphics();
 	if (m_textVertices)
 		delete m_textVertices;
+
+    if (m_textTexture) {
+		gs_texture_destroy(m_textTexture);
+    }
+
```
Comment (EddyGharbi) [inline obs-studio-server/source/nodeobs_display.cpp:368]: Please correct indentation here.

**Review by EddyGharbi — CHANGES_REQUESTED** [review]

File: obs-studio-server/source/nodeobs_display.cpp (hunk 1 above)
Comment (RodrigoHolztrattner) [reply obs-studio-server/source/nodeobs_display.cpp:368]: Done!

Comment (EddyGharbi) [reply obs-studio-server/source/nodeobs_display.cpp:368]: I still see the same issue.

Comment (RodrigoHolztrattner) [reply obs-studio-server/source/nodeobs_display.cpp:368]: I think now it's fixed

Comment (EddyGharbi) [reply obs-studio-server/source/nodeobs_display.cpp:368]: Looks good ty

**Pull request merged**